### PR TITLE
Changed flag images. Changed flags file names to ISO alpha-2 standard.

### PR DESCRIPTION
**Pull request by kasperaitis**
Changed Flag images due style and size inconsistency also some was used from freeflagicons.com which are copyrighted. Style now is more flat to reflect GUI. Changed flag file names to ISO alpha-2 standard to be more consistent except those which are not in this standard. Also reordered flag list alphabetically in qml.qrc.

### CODE REVIEW

**Comment by erciccione** [comment]
@kasperaitis please rebase

**Comment by kasperaitis** [comment]
@erciccione Rebased.

File: lang/languages.xml [hunk 1]
```diff
@@ -17,42 +17,42 @@ Deutsch		(German)
...
+        <language display_name="العربية" locale="ar_AR" wallet_language="English" flag="/lang/flags/eg.png" qs="none"/>
+        <language display_name="Slovenski" locale="sl_SI" wallet_language="English" flag="/lang/flags/si.png" qs="none"/>
+        <language display_name="Srpski" locale="rs_RS" wallet_language="English" flag="/lang/flags/rs.png" qs="none"/>
         <language display_name="Català" locale="cat_ES" wallet_language="English" flag="/lang/flags/catalonia.png" qs="none"/>
         <language display_name="Türkçe" locale="tr_TR" wallet_language="English" flag="/lang/flags/tr.png" qs="none"/>
-        <language display_name="Українська" locale="uk_UA" wallet_language="English" flag="/lang/flags/ukraine.png" qs="none"/>
+        <language display_name="Українська" locale="uk_UA" wallet_language="English" flag="/lang/flags/ua.png" qs="none"/>
         <language display_name="Lietuvių" locale="lt_LT" wallet_language="English" flag="/lang/flags/lt.png" qs="none"/>
         <language display_name="Suomi" locale="fi_FI" wallet_language="English" flag="/lang/flags/finland.png" qs="none"/>
```
Comment (erciccione) [inline lang/languages.xml]: This line should be adapted as well ot this flag won't display properly: `flag="/lang/flags/fi.png"`

File: lang/languages.xml [hunk 2]
```diff
@@ -17,42 +17,42 @@ Deutsch		(German)
...
-        <language display_name="Română" locale="ro_RO" wallet_language="English" flag="/lang/flags/romania.png" qs="none"/>
-        <language display_name="Dansk" locale="da_DK" wallet-language="English" flag="/lang/flags/denmark.png" qs="none"/>
-        <language display_name="Česky" locale="cs_CZ" wallet_language="English" flag="/lang/flags/czech.png" qs="none"/>
-        <language display_name="Slovensky" locale="sk_SK" wallet_language="English" flag="/lang/flags/slovakia.png" qs="none"/>
-        <language display_name="العربية" locale="ar_AR" wallet_language="English" flag="/lang/flags/egypt.png" qs="none"/>
-        <language display_name="Slovenski" locale="sl_SI" wallet_language="English" flag="/lang/flags/slovenia.png" qs="none"/>
-        <language display_name="Srpski" locale="rs_RS" wallet_language="English" flag="/lang/flags/srbija.png" qs="none"/>
+        <language display_name="Español" locale="es_ES" wallet_language="Español" flag="/lang/flags/es.png" qs="none"/>
+        <!-- <language display_name="Suomen kieli" locale="fi" wallet_language="English" flag="/lang/flags/fi.png" qs="none"/> -->
```
Comment (erciccione) [inline lang/languages.xml]: This line can be removed

**Comment by kasperaitis** [comment]
@erciccione Missed that one. Done.

**Pull request merged**